scripts/should-rebuild-on-vercel.sh

```diff
@@ -56,12 +56,10 @@ elif [ "$#" -gt 1 ]; then
     exit 4
 fi
 
-EXAMPLE="${1%/}"
+PROJECT="${1%/}"
```

Comment:
This change is fine, as long as you update all the existing invocations to include the `examples/` prefix in the argument.

Reply:
Unrelated, but in case you're wondering, the `"${1%/}"` syntax means: take the value of `$1` (the first argument to the script), and strip off the `/` suffix, if there is any. It means that the script will work the same whether you pass `examples/foo` or `examples/foo/` as the argument.

Reply:
> This change is fine, as long as you update all the existing invocations to include the examples/ prefix in the argument.

Yep I'll update the paths for all existing examples before merging, and update the Notion doc for future examples.